unit/java_bytecode/java_object_factory/gen_nondet_string_init.cpp

```diff
@@ -33,6 +33,20 @@ TEST_CASE("Generate string object")
   bool failed=symbol_table.add(jlo_sym);
   CHECK_RETURN(!failed);
 
+
+  // Add java.lang.Object
+  symbolt new_symbol;
```

Comment:
You shouldn't need to do this -- jlo is always in the class loader's queue

Reply:
Not sure we have a class loader here (did you see we are in a unit test?)